### PR TITLE
set progressive mode to FULL when capella is defined

### PR DESCRIPTION
**Pull request by tbenr**
change default progressive mode for Capella-enabled networks

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [ ] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java [hunk 1]
```diff
@@ -276,8 +277,16 @@ public Eth2NetworkConfiguration build() {
...
-                          capellaForkEpoch.ifPresent(capellaBuilder::capellaForkEpoch));
+                      capellaBuilder -> {
+                        capellaForkEpoch.ifPresent(capellaBuilder::capellaForkEpoch);
+
+                        // set progressiveBalancesMode to FULL for all milestones if capella is
+                        // defined and no commandline override has been specified
+                        if (!FAR_FUTURE_EPOCH.equals(capellaBuilder.getCapellaForkEpoch())
+                            && progressiveBalancesMode == DEFAULT_PROGRESSIVE_BALANCES_MODE) {
+                          builder.progressiveBalancesMode(ProgressiveBalancesMode.FULL);
+                        }
```
Comment (rolfyone) [inline ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java]: we could just go all in and skip the if statement, no-one else has a fallback option as far as I can tell...

Comment (rolfyone) [reply ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java]: I guess my reasoning here is the default is the one we'd want to fall back to and this logic wouldn't allow that anyway.... so the CLI flag is almost not allowing us to do the only option we'd want...

Comment (tbenr) [reply ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java]: > .... so the CLI flag is almost not allowing us to do the only option we'd want...

Yep true. Going to remove it..

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**